test/Cache.spec.ts

```diff
@@ -1,15 +1,14 @@
 import * as fs from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
-
 import { Cache } from '../src/Cache';
 
 describe('Cache', () => {
   let cacheDir: string;
   let cache: Cache;
 
-  const dummyUrl = 'dummy://';
-  const sanitizedDummyUrl = 'dummy';
+  const dummyUrl = 'dummy://dummypath';
+  const sanitizedDummyUrl = Cache.getCacheDirectory(dummyUrl);
```

Comment:
I think this value is supposed to be hardcoded, since it's testing the implementation.